Rx.xcodeproj/project.pbxproj

```diff
@@ -3893,6 +3893,7 @@
 		A2897D50225CA1E7004EA481 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				BUILD_LIBRARY_FOR_DISTRIBUTION = NO;
```

Comment:
This is now disabled for Debug and Release-Test configurations, and enabled for the release configuration, doesn't seem like a good idea to generate swiftinterfaces from those anyway